lib/web_console/templates/console.js.erb

```diff
@@ -370,4 +382,35 @@ REPLConsole.prototype.scrollToBottom = function() {
   this.inner.scrollTop = this.inner.scrollHeight;
 };
 
+/**
+ * Install the console into the element with a specific ID.
+ * Example: REPLConsole.install_into("target-id")
+ */
+REPLConsole.install_into = function(id) {
```

Comment:
Can we name this in `camelCase` to follow the convention in the rest of the file? Also I think we may extract a common method for [this](https://github.com/rails/web-console/blob/9ef07c7d546d9820fd510a271d40cf687636bd22/lib/web_console/templates/error_page.js.erb#L27-L42) functionality as well. That way we keep the information about "console" things in one place only.